enforcer-rules/src/site/apt/requireFilesDontExist.apt.vm

```diff
@@ -26,7 +26,10 @@
 Require Files Don't Exist
 
   This rule checks that the specified list of files do not exist.
-
+  
+  The mounted filesystem(s) will dictate the case-sensitivity rules when checking files. If checks are required that
```

Comment:
delete "mounted" and consider getting rid of the parentheses or just changing this to "The filesystem" as is every file is on a single filesystem.

Reply:
Done